README.md

```diff
@@ -1,46 +1,54 @@
 ## IES Syntax Highlighter
 
-The IES Syntax Highlighter is a Visual Studio Code (VS Code) extension designed to make working with IES in [Terse RDF Triple Language (TTL)](https://www.w3.org/TR/turtle/) (.ttl) files easier. This extension was created initially for internal use within the Data Science team at Telicent to make working with IES in TTL files nicer.
+A [Visual Studio Code](https://code.visualstudio.com) (VSCode) extension to help working with [IES](https://github.com/dstl/IES4) in [Terse RDF Triple Language (TTL)](https://www.w3.org/TR/turtle/) (.ttl) files
+
+Features:
+<ul>
+  <li>
+    <strong>Colour highlighting</strong> - Highlights IES classes according to their defined colours<br>
+    <img src="./images/Overview.png" alt="Highlights IES classes according to their defined colours" style="height:300px">
+  </li>
+  <li>
+    <strong>Diagnostics</strong> - Syntax highlights missing commas, dots, semicolons<br>
+    <img src="./images/Diagnostics.png" alt="Syntax highlights missing commas, dots, semicolons">
+  </li>
+  <li>
+    <strong>IES class suggestions</strong> - Suggests IES classes as you type<br>
+    <img src="./images/Suggestions.jpeg" alt="Suggests IES classes as you type" style="height:300px">
+  </li>
+  <li>
+    <strong>On hover matches</strong> - Hover over a URI to reveal IES type<br>
+    <img src="./images/OnHover.png" alt="Hover over a URI to reveal IES type">
+  </li>
+</ul>
 
-More information about IES can be found here: [IES4 GitHub repository](https://github.com/dstl/IES4)
 
-### Features
 
-- **Colour highlighting:** Highlights IES classes according to their defined colours
-- **Diagnostics:** Syntax highlights missing commas, dots, semicolons
-- **Autocomplete Classes:** Suggests IES classes as you type
-- **On-hover reveal URI type** Hover-over a URI to reveal its type
 
-#### Color highlighting
-![colour highlighting screenshot](./images/Overview.png)
-#### Diagnostics
-![syntax diagnostics screenshot](./images/Diagnostics.png)
-#### IES class suggestions
-![IES class suggestion screenshot](./images/Suggestions.jpeg)
-#### On hover matches
-![On-hover IES type tracking](./images/OnHover.png)
 
 ## Install
 
-1. Download the latest release from: [IES Syntax Highlighter releases](https://github.com/Telicent-io/ies-ext/releases)
+1. Download latest [/releases](/releases)
```

Comment:
I think this is a broken link: I couldn't see the releases folder.

Reply:
Yeah. I think it won't work from within the PR. I'll need to merge to main and check. 

If it is still broken, I'll need to do.... wait I can test on a fork

Reply:
Tested on my personal branch and it didn't work. Fixed.